### ISSUE TITLE
Python base packages don't provide anything ...

### ISSUE THREAD

**Issue by Begasus** [haikuports/haikuporter#192]
I've seen it passing by while working on some of the different python_version packages, python versions are defined and they build ok.
You get 4 different packatges, 1 base (empty), and one for each python version (2.7, 3.6 and 3.7)
Is there a way to not build the base package or set it to 2.7 as a base and 3.6 and 3.7 as second and third package?

**Comment by pulkomandy** [comment]
Why would 2.7 be the base? Python 2.7 is deprecated and should not be the default. If you want to make one of these the base, use 3.7. But then we're in for a lot of renaming when 3.8 comes in.

Removing the base package would be nice, we should add that to Haikuporter if not already possible.

**Comment by Begasus** [comment]
2.7 was the first that came to mind here :)

**Comment by korli** [comment]
Proposal: DISABLE_MAIN_PACKAGE="yes"

**Comment by alaviss** [comment]
An another way is to make them a meta package, so people installing them will get all `_python*` variants.

**Comment by Begasus** [comment]
Both valid options, but we need one way to go with this :)

**Comment by Glutexo** [comment]
Sorry for the dumb question. How to install Python 3 when Haikuporter requires Python 3 for running? With default Haiku Beta 2, only Python 2.7 is available and running _haikuporter_ results in

```
env: 'python3': No such file or directory
```

**Comment by pulkomandy** [comment]
`pkgman install cmd:python3` should do it

**Comment by Glutexo** [comment]
You are right! Thank you for your help! 🙇🏻‍♂️ Hopefully this helps anybody else with the same problem.

**Comment by Begasus** [comment]
Some "main" packages are not empty (for instance pytest, it provides the binary in the main package, for this one I also saw that it uses python 3.7 modules, so maybe that one should be dimmed to be only build with 3.7 (pushed a version earlier, but my skills on python are not that great)) :)

**Comment by jmairboeck** [comment]
This would probably also solve the name conflict between [python pathtools](https://github.com/haikuports/haikuports/blob/master/dev-python/pathtools/pathtools-0.1.2.recipe) and [perl pathtools](https://github.com/haikuports/haikuports/blob/master/dev-perl/pathtools/pathtools-3.75.recipe) nicely when the python one doesn't build a main package any more.
